css/styles.css

```diff
@@ -114,6 +119,18 @@ label {
   top: 10%;
 }
 
+.abilities-container {
+  font-size: clamp(0.05rem, 2vw, 0.7rem);
```

Comment:
How cool! Never seen `clamp` before, but it seems to do the trick! I would've gone with media queries, but I'll definitely start using this myself!